### DISCUSSION TITLE
missing search box for base branch when opening pull request

### DISCUSSION THREAD

**Discussion by vpop-dn** [community/community#129286] [Pull Requests]
### when openning a PR, choosing a base branch had a search box, from few days is missing

is this a bug, or a feature ?
anybody knows if this is a user setting, that could I accidentally changed ?

**Comment by dheeresh-attentive** [comment]
Looks like a goddamn bug, only a hardcore masochist will disable it by default

**Comment by dazlee** [comment]
<img width="825" alt="Screen Shot 2024-06-21 at 5 21 40 PM" src="https://github.com/community/community/assets/499822/d6484894-f3e6-409d-a72b-f7d73461f0fe">

<img width="806" alt="Screen Shot 2024-06-21 at 5 24 50 PM" src="https://github.com/community/community/assets/499822/0503b642-19d9-4254-917e-d1ba4ad95b39">


for workaround, just drag SelectMenu-filter and drop upper <tab-container....

**Reply by harrisonlin0217** [reply]
crazy man, unbelivable, it works for me!

**Reply by Subaruuuu** [reply]
thx man, it help a lot

**Reply by TuxIbears** [reply]
crazy man, it saves our team!

**Comment by Chris1245678** [comment]
It seems the search box for choosing a base branch when opening a pull request (PR) is missing. This might be due to a recent update or a temporary issue. Try refreshing your browser or clearing your cache. If the problem persists, check if others in your team are experiencing the same issue and consider reporting it to the support team of the platform you are using. They may be able to provide a fix or further guidance. For more information [Click here](https://www.roofersgreenvillenc.com/).

**Comment by vpop-dn** [comment]
tried again, fixed.
thanks
